includes/event/event-fields.php

```diff
@@ -8,23 +8,23 @@ function event_custom_fields()
         'event_schedule',
         __('Event schedule', 'agenda-core'),
         'event_schedule_callback',
-        'event'
+        'events'
     );
 
     // Add meta box for event venue
     add_meta_box(
         'event_venue',
         __('Event venue', 'agenda-core'),
         'event_venue_callback',
-        'event'
+        'events'
     );
 
     // Add meta box for event organizer
     add_meta_box(
         'event_organizer',
-        __('Event organizer', 'agenda-core'),
+        __('Event organisation', 'agenda-core'),
```

Comment:
I'm totally onboard with renaming the `organizer` CPT to `organization` to make more sense. However the `Event organizer` should remain the same since an Event should have an `organizer` who organizes the event (which then technically refers to an `organization`)

Reply:
```suggestion
        __('Event organizer', 'agenda-core'),
```